### PR TITLE
Merge levelbuilder to staging 20240610

### PR DESCRIPTION
**Pull request by sureshc**
Resolve merge conflict that prevented automated merge of `levelbuilder` branch into `staging` branch from completing on Mon 12:30 AM PDT.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -230,12 +230,6 @@ img.video_thumbnail {
     }
   }
 
-  @media screen and (max-width: 1120px) {
-    .create_menu {
-      display: none !important;
-    }
-  }
-
```
Comment (davidsbailey) [inline dashboard/app/assets/stylesheets/application.scss]: please revert the change to this file. Kelby's most recent change to this file adds this code back, and I don't think we should be letting any non-content changes through the DTS anyway.

Comment (sureshc) [reply dashboard/app/assets/stylesheets/application.scss]: Thanks for catching that. Looks like I should also revert the change to `shared/css/user-menu.scss`?

Comment (kelbyhawn) [reply dashboard/app/assets/stylesheets/application.scss]: @sureshc correct, and thanks for noticing this @davidsbailey!

**Review by kelbyhawn — APPROVED** [review]

**Review by davidsbailey — APPROVED** [review]
